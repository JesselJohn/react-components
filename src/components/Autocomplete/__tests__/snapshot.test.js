import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import TestComponent from "../test.component";

describe("Root Component Tests", () => {
  it("Matches Shallow SnapShot", done => {
    expect(shallowToJson(shallow(<TestComponent />))).toMatchSnapshot();
    done();
  });

  it("Matches Rendered SnapShot", done => {
    expect(renderer.create(<TestComponent />).toJSON()).toMatchSnapshot();
    done();
  });
});
