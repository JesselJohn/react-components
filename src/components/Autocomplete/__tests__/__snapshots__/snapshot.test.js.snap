// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Component Tests Matches Rendered SnapShot 1`] = `
<div
  className="nwc-autocomplete "
>
  <input
    checked={null}
    className="nwc-inp  "
    disabled={false}
    id="autocomplete-input"
    maxLength={null}
    name={null}
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    placeholder="Enter text"
    tabIndex="0"
    type="text"
    value=""
  />
</div>
`;

exports[`Root Component Tests Matches Shallow SnapShot 1`] = `
<Autocomplete
  className=""
  getSelection={[Function]}
  inpList={
    Array [
      "Arrow",
      "Levis",
      "Gap",
    ]
  }
  minTextLength={0}
  renderInput={[Function]}
  renderList={[Function]}
  selectedIndex={0}
/>
`;
